picard/ui/ui_options_interface_theme.py

```diff
@@ -1,50 +0,0 @@
-# -*- coding: utf-8 -*-
-
-# Automatically generated - don't edit.
-# Use `python setup.py build_ui` to update it.
-
-from PyQt5 import QtCore, QtGui, QtWidgets
-
-
```

Comment:
Why is this file removed?

Reply:
It's a leftover from when you renamed it to "colors" :)

Reply:
Oh well... ;)